src/components/loader-bar/loader-bar.style.js

```diff
@@ -24,20 +24,20 @@ const innerBarAnimation = keyframes`
 `;
 
 const StyledLoaderBar = styled.div`
-  ${({ size, theme }) => css`
+  ${({ size }) => css`
```

Comment:
Remove `import baseTheme from "../../style/themes/base";`

Reply:
StyledLoader uses ${margin} so it's needed.

StyledLoader.defaultProps = {
  theme: baseTheme,
};

Reply:
You're right I didn't notice that